### DISCUSSION TITLE
Server Error

### DISCUSSION THREAD

**Discussion by KaueASB** [community/community#35373] [Other Feature Feedback, Questions, & Ideas]
Hello,

I'm getting error 500 when trying to access my profile while logged in. I am able to access the repositories, including the URL correctly, but the profile is not being possible. The strange thing is that I can see it normally when I'm logged out.

**Comment by Bakugo90** [comment]
I have the same problem. i have also ask for github assitance.

**Reply by KaueASB** [reply]
How long ago did you requested?

**Reply by Bakugo90** [reply]
30 min ago

**Reply by Bakugo90** [reply]
I can see your profile on my side

**Reply by KaueASB** [reply]
I can't see yours!

**Reply by radar-blip** [reply]
@KaueASB Yours seems to be working now. @Bakugo90 No dice.

**Reply by radar-blip** [reply]
Same issue here: #35362

**Reply by KaueASB** [reply]
@linuxlawson I checked and the error persists!

**Reply by Bakugo90** [reply]
> @KaueASB Yours seems to be working now. @Bakugo90 No dice.

So How to solve this problem ?

**Reply by radar-blip** [reply]
Interesting. I can see yours.

**Reply by KaueASB** [reply]
@linuxlawson I checked again and now it's back to normal. Ty

**Reply by radar-blip** [reply]
@Bakugo90 Yours is back up now.

**Reply by Bakugo90** [reply]
> @Bakugo90 Yours is back up now.

I just saw it. thank you boss

**Reply by radar-blip** [reply]
Anytime. All seem to be fixed now.

**Comment by stallardjw** [comment]
I am also experiencing this issue.

**Reply by KaueASB** [reply]
This is very bad my friend

**Reply by Bakugo90** [reply]
> I am also experiencing this issue.

I think that in a few minutes, everything will be back to normal on your side.

**Comment by 3kh0** [comment]
This happens every so often, here are a few quick fixes that sometimes help.

- Reload the page (easiest in the book)
- Reload and clear cache, in chrome do ctrl+shift+r
- Disconnect from wifi, and reconnect
- Open cmd and do `ipconfig/release` and then `ipconfig/renew`
- Turn off your computer, wait 10 seconds, turn it back on.
- Try a different browser
- Clear cookies and site data from github.com
- Reboot your router

These are some quick and easy fixes, this issue happens from time to time again, and these fixes seems to fix it for me.

*If this helped please mark this as the answer!* ❤️
